### ISSUE TITLE
CI for running plugin verifier every push

### ISSUE THREAD

**Issue by ericdallo** [clojure-lsp/clojure-lsp-intellij#10]

**Comment by matuella** [comment]
Just wanted to raise some questions:
- Can we use the GH actions for this? Do we have some sort of limitation?
- I'm not sure if I got the verifier part straight, but would it consist of running [intellij-plugin-verifier](https://github.com/JetBrains/intellij-plugin-verifier) against this plugin? I'm not familiar with IntelliJ plugins, so I would like understand the objective for this check.

**Comment by ericdallo** [comment]
> Can we use the GH actions for this?

Absolutely, public repos can use that, [clojure-lsp CI](https://github.com/clojure-lsp/clojure-lsp/) uses a lot github CI for tests and release

> but would it consist of running [intellij-plugin-verifier](https://github.com/JetBrains/intellij-plugin-verifier) against this plugin?

Yep, unfortunately, there are some false positives given its Clojure integration in the plugin, but we could check if the output of plugin verifier is the same or it's not increasing

This is the result of plugin verifier for the current release 0.1.4
![image](https://github.com/clojure-lsp/clojure-lsp-intellij/assets/7820865/df57cf32-cb22-4313-94f5-2888f960ab3e)
